packages/front-end/components/SafeRollout/DecisionHelpText.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  getHealthSettings,
+  getSafeRolloutDaysLeft,
+  getSafeRolloutResultStatus,
+} from "shared/enterprise";
+
+import { SafeRolloutRule } from "back-end/src/validators/features";
+import HelperText from "@/components/Radix/HelperText";
+import { useUser } from "@/services/UserContext";
+import { useSafeRolloutSnapshot } from "@/components/SafeRollout/SnapshotProvider";
+
+const DecisionHelpText = ({ rule }: { rule: SafeRolloutRule }) => {
+  const {
+    safeRollout,
+    snapshot: snapshotWithResults,
+  } = useSafeRolloutSnapshot();
+
+  const { hasCommercialFeature, organization } = useUser();
+  const settings = organization?.settings;
+
+  const numberFormatter = Intl.NumberFormat();
+  const percentFormatter = Intl.NumberFormat(undefined, {
+    style: "percent",
+    minimumFractionDigits: 2,
+    maximumFractionDigits: 2,
+  });
+
+  if (!safeRollout || !safeRollout.startedAt || rule.enabled === false) {
+    return null;
+  }
+
+  const daysLeft = getSafeRolloutDaysLeft({
+    safeRollout,
+    snapshotWithResults,
+  });
+
+  const decisionStatus = getSafeRolloutResultStatus({
+    safeRollout,
+    healthSettings: getHealthSettings(
+      settings,
+      hasCommercialFeature("decision-framework")
+    ),
+    daysLeft,
+  });
+
+  // If the safe rollout has been rolled back or released, explain that the safe rollout is
+  // acting as a temporary rollout with the control or variation value
+  if (safeRollout.status === "rolled-back") {
+    return (
+      <HelperText status="info" mb="3">
+        All users will receive the control value. If this is no longer needed,
+        you can disable or delete this rule.
+      </HelperText>
+    );
+  } else if (safeRollout.status === "released") {
+    return (
+      <HelperText status="info" mb="3">
+        All users will receive the variation value. If this is no longer needed,
```

Comment:
nit: We refer to it as `Rollout` value in the `SERVE` section, should we also use the same here?